Dockerfile

```diff
@@ -62,10 +63,12 @@ RUN set -ex \
     && pip install pyasn1 \
     && pip install google-api-python-client \
     && pip install pandas-gbq \
-    && pip install https://github.com/apache/incubator-airflow/archive/1.9.0.tar.gz \
+    && pip install -U setuptools \
+    && pip install https://github.com/apache/incubator-airflow/archive/1.10.1.tar.gz \
```

Comment:
@smpiano instead I would use the variable instead of having the version hardcoded in here: `&& pip install https://github.com/apache/incubator-airflow/archive/${AIRFLOW_VERSION}.tar.gz`